recipes-kernel/linux/files/iot2050_defconfig_base

```diff
@@ -490,11 +470,9 @@ CONFIG_SND_SOC_DAVINCI_MCASP=y
 CONFIG_SND_SOC_AK4613=m
 CONFIG_SND_SOC_PCM3168A_I2C=m
 CONFIG_SND_SOC_TLV320AIC31XX=m
-CONFIG_SND_SOC_TLV320AIC3X=m
```

Comment:
This driver was split into I2C and SPI attached ones. We may have used the I2C version before.

Reply:
...or this was simply inherited from the EMV defconfig which has extensions we don't have.